grakn-engine/src/main/java/ai/grakn/engine/tasks/config/ConfigHelper.java

```diff
@@ -18,7 +18,7 @@
 
 package ai.grakn.engine.tasks.config;
 
-import ai.grakn.engine.tasks.TaskId;
+import ai.grakn.engine.TaskId;
```

Comment:
But the same package as `TaskId`! Something probably needs to change, but I don't know which.